app/build.gradle

```diff
@@ -86,6 +88,7 @@ dependencies {
 
     implementation 'androidx.core:core-ktx:1.9.0'
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
+    implementation "androidx.compose.material3:material3:1.0.1"
```

Comment:
Api compose core?

Reply:
I could, it currently isn't bringing in uiCore and it is an application gradle file so I didn't see a reason to api the material3. Would it be better to just bring in uiCore as that is where the api'd material 3 is currently?

Reply:
you know, I was completely mistakeing it IS in the compose core not ui core. updated.